db/pgstorage/pgstorage.go

```diff
@@ -505,3 +534,10 @@ func (p *PostgresStorage) ResetTrustedState(ctx context.Context, batchNumber uin
 	_, err := p.getExecQuerier(dbTx).Exec(ctx, resetTrustedStateSQL, batchNumber)
 	return err
 }
+
+// UpdateBlocks updates the hash of blocks.
+func (p *PostgresStorage) UpdateBlocks(ctx context.Context, networkID uint, blockNum uint64, dbTx pgx.Tx) error {
```

Comment:
Is this function used only for testing purposes right? Maybe we coan call it UpdateBlocksForTesting or something like that